xbmc/addons/kodi-addon-dev-kit/include/kodi/libXBMC_addon.h

```diff
@@ -563,7 +563,7 @@ namespace ADDON
     * @brief Adds options to the curl file created with CURLCeate
     * @param file file pointer to the file returned by CURLCeate
     * @param type option type to set
-    * @param name name of the option
+    * @param name name of the option (see https://github.com/xbmc/xbmc/blob/master/xbmc/filesystem/CurlFile.cpp#L793)
```

Comment:
sooner or later, this will link to a wrong line

Reply:
Thats true, but the dev should still be able to find the required information. :) But yes, I will remove it again.

Reply:
-1 for this, document it where it belongs, please.  We never do it like this for a reason.